CONTRIBUTING.md

```diff
@@ -118,3 +118,11 @@ def test_elasticsearch_custom_fields(elasticsearch_fixture):
     document_store = ElasticsearchDocumentStore(index="haystack_test_custom", text_field="custom_text_field",
                                                 embedding_field="custom_embedding_field")
 ``` 
+
+## Contributor Licence Agreement (CLA)
+
+Significant contributions to Haystack require a Contributor License Agreement (CLA). CLAs are quite common among company backed open-source frameworks and our CLA’s wording is similar to other popular projects, like [Rasa](https://cla-assistant.io/RasaHQ/rasa) or [Google's Tensorflow](https://cla.developers.google.com/clas/new?domain=DOMAIN_GOOGLE&kind=KIND_INDIVIDUAL) (retrieved 4th November 2021). 
```

Comment:
I realized we should also write that we will get in contact if we shall deem the contribution "significant".

... require a Contributor License Agreement (CLA). If the contribution requires a CLA we will get in contact with you.

Furthermore please make the "and anyone" in bold. I think it is a cool part of our CLA and should be emphasized :D

Reply:
One FAQ type page explaining in EL10 type about this CLA would be great. As this CLA is very open and supportive to the contributors hence new contributors should not be spooked by this. Hence explaining it in FAQ would be great to answer questions like "if work in some company and contribute, how is it going to affect me?" etc many more.